### PR TITLE
Add 'changes' to this article

### PR DESCRIPTION
**Pull request by Girrajjangid**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/python/itertools/index.md [hunk 1]
```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
 combinations_with_replacement('ABCD', 2)	 	                                                     AA AB AC AD BB BC BD CC CD DD
 
 
-Source:https://docs.python.org/3/library/itertools.html
\ No newline at end of file
+Source:https://docs.python.org/3/library/itertools.html
```
Comment (Manish-Giri) [inline guide/english/python/itertools/index.md]: `Source` should be at the end of the file.

File: guide/english/python/itertools/index.md [hunk 2]
```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
...
+
+##Changes
```
Comment (Manish-Giri) [inline guide/english/python/itertools/index.md]: Please remove this line.

File: guide/english/python/itertools/index.md [hunk 3]
```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
...
+
+###compress()
+It make an iterator that filters elements from data returning only those that have a corresponding element in selectors that evaluates to True. Stops when either the data or selectors iterables has been exhausted.
+
+import itertools as i
```
Comment (Manish-Giri) [inline guide/english/python/itertools/index.md]: import itertools as i
 i.compress('ABCDEF', [1,0,1,0,1,1]) 
 --> A C E F

Please use correct formatting for posting code samples and output. Use the [guide formatting](https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/8e8ea8cbdffc7f8bcf2c846824993c46ff8b0ae9/guide/english/python/itertools/index.md) as a reference.

File: guide/english/python/itertools/index.md [hunk 4]
```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
...
+i.compress('ABCDEF', [1,0,1,0,1,1]) 
+--> A C E F
+
+Sourse : https://docs.python.org/2/library/itertools.html
```
Comment (Manish-Giri) [inline guide/english/python/itertools/index.md]: `Sourse` is incorrect. Please use the existing `Source:https://docs.python.org/3/library/itertools.html` that links to Python 3.

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Changes to be addressed.

File: guide/english/python/itertools/index.md (hunk 3 above)
Comment (Girrajjangid) [reply guide/english/python/itertools/index.md]: Thanks for the suggestion. I Will keep in mind next time.

**Review by Manish-Giri — APPROVED** [review]

**Review by Manish-Giri — APPROVED** [review]
Hi @Girrajjangid 

Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍 We're happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**